core/options.js

```diff
@@ -97,6 +97,10 @@ Blockly.Options = function(options) {
   if (hasCss === undefined) {
     hasCss = true;
   }
+  var hasModal = options['modal'];
+  if (hasModal === undefined) {
```

Comment:
Is this meant to be a callback to a modal then? If that's the case we need to make sure the interface is clearly defined.